coffee/factory/user.coffee

```diff
@@ -4,9 +4,1390 @@ User = (apiService, messageService, sortService) ->
 
     @initialized = false
     _repos = null
+    _colors = {
```

Comment:
instead of putting the colors here,
why not use the angular.constant type instead?
```
angular.constant('Colors', {
....
})
```
then in the `ListItemCtrl `, you can simply inject this `.constant` 
`ListItemCtrl = ($scope, githubService, Colors) ->`
this way you dont need to call this `$scope.globalState.user.getColors()` function